src/python/nimbusml/internal/core/base_pipeline_item.py

```diff
@@ -387,8 +390,11 @@ def __getstate__(self):
     def __setstate__(self, state):
         "Restore a pickled object."
         for k, v in state.items():
-            if k not in {'modelbytes', 'type'}:
+            if k not in {'modelbytes', 'type', 'export_version'}:
```

Comment:
can u rebase ? this change should be in master already

Reply:
I did, but the commit history in this branch got screwed up while fixing conflicts. This (and other changes) aren't actually in this PR.